docker/openemr/flex-edge/openemr.sh

```diff
@@ -85,27 +97,26 @@ auto_setup() {
 # - false for the Kubernetes startup job and manual image runs
 AUTHORITY=yes
 OPERATOR=yes
-if [ "$K8S" == "admin" ]; then
-    OPERATOR=no
-elif [ "$K8S" == "worker" ]; then
-    AUTHORITY=no
-fi
+case "${K8S}" in
+    admin) OPERATOR=no;;
+    worker) AUTHORITY=no;;
+    *) :;;
+esac
 
-if [ "$SWARM_MODE" == "yes" ]; then
+if [ "${SWARM_MODE}" = yes ]; then
     # atomically test for leadership
     set -o noclobber
-    { > /var/www/localhost/htdocs/openemr/sites/docker-leader ; } &> /dev/null || AUTHORITY=no
+    { : > /var/www/localhost/htdocs/openemr/sites/docker-leader ; } > /dev/null 2>&1 || AUTHORITY=no
     set +o noclobber
-    
-    if [ "$AUTHORITY" == "no" ] &&
-       [ ! -f /var/www/localhost/htdocs/openemr/sites/docker-completed ]; then
-        while swarm_wait; do
+
+    if [ "${AUTHORITY}" = "no" ]; then
```

Comment:
shouldn't the quotes be removed from the no above (noted this pattern of change elsewhere)?

Reply:
seeing lots of other places below where quotes is not removed. maybe i am missing something?

Reply:
If this were bash, then we would use `[[ ... ]]` and remove the quotes. But here in POSIX sh we want the quotes, because they prevent improper expansions that will make the test fail.

Reply:
hi, but then why were the quotes removed above in lines 81 and 106?

Reply:
Ah, I removed the quotes around bare words with no special characters because I think they're less ambiguous that way. It wouldn't hurt to leave them in single quotes, although I do think double quotes should be reserved for when they're actually necessary.